public/components/register_model/register_model.tsx

```diff
@@ -231,6 +231,7 @@ export const RegisterModelForm = ({ defaultValues = DEFAULT_VALUES }: RegisterMo
           // TODO: Fill model format here
           const { config, url } = preTrainedModel;
           form.setValue('modelURL', url);
+          form.setValue('modelFileFormat', 'TORCH_SCRIPT');
```

Comment:
How about separate a variable shared in `.getPreTrainedModel$` and `form.setValue`. We can use `variable.toUpperCase()` here

Reply:
Just want to refactor code like these:
```tsx
    const preTrainFormat = "torch_script";
    const subscriber = modelRepositoryManager
      .getPreTrainedModel$(nameParams, preTrainFormat)
      .subscribe(
        (preTrainedModel) => {
          // TODO: Fill model format here
          const { config, url } = preTrainedModel;
          form.setValue('modelURL', url);
          form.setValue('modelFileFormat', preTrainFormat.toUpperCase());
```
It's not a critical issue, we can update in the future.